src/components/Nav/NavItem.react.js

```diff
@@ -15,53 +16,99 @@ type Props = {|
   +hasSubNav?: boolean,
   +onClick?: () => void,
   +active?: boolean,
+  +subItems?: React.ChildrenArray<React.Element<typeof Nav.SubItem>>,
+  +subItemsObjects?: Array<{|
```

Comment:
[nit] You can probably export this type (if it is the same as used above) and then import it here so we don't have redundant definitions of the same type